frontend/src/redux/roadmaps/selectors.ts

```diff
@@ -79,3 +79,18 @@ export const chosenIntegrationSelector = (name: string) =>
   createSelector(chosenRoadmapSelector, (roadmap) =>
     roadmap?.integrations.find((it) => it.name === name),
   );
+
+export const roadmapsVersionsSelector = (roadmapId?: number) =>
+  roadmapId
+    ? createSelector(idRoadmapSelector(roadmapId), (roadmap) => {
+        if (!roadmap?.versions) return undefined;
+        return [...roadmap?.versions].sort(
```

Comment:
The condition could be moved to just select the input selector, so the additional processing need not be duplicated in the code.

(Also the `?.` is unnecessary after the check)
```suggestion
    createSelector(roadmapId ? idRoadmapSelector(roadmapId) : chosenRoadmapSelector, (roadmap) => {
        if (!roadmap?.versions) return undefined;
        return [...roadmap.versions].sort(
```

Reply:
That is really neat - updated to include the changes!